crypto/cms/cms_env.c

```diff
@@ -909,20 +977,20 @@ BIO *cms_EnvelopedData_init_bio(const CMS_ContentInfo *cms)
 
     /* If error or no cipher end of processing */
 
-    if (!ret || !ec->cipher)
+    if (!ret)
```

Comment:
not sure why the cipher check was removed - but if the new check is correct then the comment should get updated.

Reply:
Restored. Thanks!

Reply:
Updated the comment.